src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -421,3 +424,83 @@ def prepare_logs_argument(c):
 
     with self.argument_context('spring-cloud service-registry unbind') as c:
         c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
+
+    with self.argument_context('spring-cloud application-configuration-service bind') as c:
+        c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
+
+    with self.argument_context('spring-cloud application-configuration-service unbind') as c:
+        c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
+
+    for scope in ['spring-cloud application-configuration-service git repo add',
+                  'spring-cloud application-configuration-service git repo update']:
+        with self.argument_context(scope) as c:
+            c.argument('patterns',
+                       help='Required patterns used to search in Git repositories. '
+                            'For each pattern, use format like {application} or {application}/{profile} '
+                            'instead of {application}-{profile}.yml, and separate them by comma.',
+                       validator=validate_acs_patterns),
+            c.argument('uri', help="Required Git URI.", validator=validate_git_uri),
+            c.argument('label', help="Required branch name to search in the Git repository."),
+            c.argument('search_paths', help='search_paths of the added config, use , as delimiter for multiple paths.')
+            c.argument('username', help='Username of the added config.')
+            c.argument('password', help='Password of the added config.')
+            c.argument('host_key', help='Host key of the added config.')
+            c.argument('host_key_algorithm', help='Host key algorithm of the added config.')
+            c.argument('private_key', help='Private_key of the added config.')
+            c.argument('host_key_check', help='Strict_host_key_checking of the added config.')
```

Comment:
Could you please explain `Strict_host_key_checking` more specifically?

Reply:
Added explanation